3rdparty/python/requirements.txt

```diff
@@ -38,6 +38,7 @@ types-setuptools==62.6.1
 types-toml==0.10.8
 typing-extensions==4.3.0
 mypy-typing-asserts==0.1.1
+node-semver==0.9.0
```

Comment:
I asked about this in slack https://pantsbuild.slack.com/archives/C0D7TNJHL/p1678052689358149